docs/UserGuide.md

```diff
@@ -3,112 +3,162 @@ layout: page
    ![Ui](images/Ui.png)
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing **| Enter &#9166; |** will open the help window.<br>
+5. Type the command in the command box and press Enter to run it. e.g. typing **`help`** and pressing **| Enter &#9166; |** will open the help window.<br>
    Some example commands you can try:
 
-   * `list` : Lists all contacts.
+    * `list` : Lists all contacts.
 
-   * `add name:John Doe phone:98765432 email:johnd@example.com address:John street, block 123, #01-01` : Adds a contact named `John Doe` to OH.
+    * `add name:John Doe phone:98765432 email:johnd@example.com address:John street, block 123, #01-01` : Adds a contact named `John Doe` to OH.
 
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
+    * `delete 3` : Deletes the 3rd contact shown in the current list.
 
-   * `clear` : Deletes all contacts.
+    * `clear` : Deletes all contacts.
 
-   * `exit` : Exits the app.
+    * `exit` : Exits the app.
 
-6. Refer to the [Features](#features) below for details of each command.
+6. Refer to the [Features](#2-features) below for details of each command.
 
---------------------------------------------------------------------------------------------------------------------
 
-## Features
+# 2. Features
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the command format:**<br>
 
-* Words :warning: enclosed between diamond brackets `<>`, are the parameters to be supplied by the user.<br>
-  e.g. in `add name:<name>`, `<name>` is a parameter which can be used as `add name:John Doe`.
+* Words enclosed between diamond brackets `<>`, are the information to be entered by the user.<br>
+  e.g. in `add name:<name>`, `<name>` is which you enter the name information, so `add name:John Doe`, will enter exactly
+  `John Doe` as the name, including the space in between.
 
-* Space can be added between the prefix and the word.<br>
+* Space can be added between the keyword and the word in `<>` bracket.<br>
   e.g. either `name: <name> phone: <phone number>` or `name:<name> phone:<phone number>` is acceptable.
 
-* Any component of a command surrounded in `[]` are optional.<br>
-  e.g. the command (`add`)[#adding-a-person-add] has an optional
-  `[department:<department>]` field, so it can be invoked with or without a
-  department.
+* Any component of a command surrounded in `[]` bracket are optional.<br>
+  e.g. the command [`add`](#22-adding-a-person--add) has an optional `[department:<department>]` part, so it can be 
+  executed with or without a department.
 
-* Any component with `...` after it means that it can be specified multiple times after the first one.
-  e.g. In [`delete`](#deleting-a-person--delete), `<id>...` means that at least one id should be specified. In [`tag`](#tagging-a-contact--tag), `[tag:<tag>]...` means that zero or more tags should be specified.
```

Comment:
I remember that there was an issue raised about this. However, I think some indication of "There can be zero or more" should still be there. Tag is the only command with that iirc.

Reply:
There's add as well now that I think about it.

Reply:
The add command does have zero or more tags, I think the tag command has to one or more